packages/env/lib/env.js

```diff
@@ -70,6 +71,20 @@ module.exports = {
 					.join( '\n' );
 		};
 
+		const downloader = ( source ) =>
```

Comment:
Function names should start with a verb. `getSourceDownloder`?

Reply:
done!